### ISSUE TITLE
opam file should specify omd < 2.0.0

### ISSUE THREAD

**Issue by rmn30** [rems-project/sail#128]
I tried an opam upgrade and couldn't compile sail due to a change in the interface of omd library. I think the opam file should specify < 2.0.0 to avoid this.

**Comment by arichardson** [comment]
It might be good to try and get sail to work with omd 2.0.0. I believe there are some workarounds in sail-cheri-riscv for bad parsing of markdown that should hopefully be fixed in that new release. But until then the version should probably be fixed.

**Comment by Alasdair** [comment]
The omd 2.0.0 version is currently marked as alpha1, so maybe it's best to wait a while in case it changes again.

I added the upper version bound for the time being

**Comment by Alasdair** [comment]
I will probably have to push a point release to opam for this I think so I'll leave this open till I do that

**Comment by Alasdair** [comment]
I added the upper bound and it's still choosing `installed omd.2.0.0~alpha1` during the build. I have no idea why

**Comment by rmn30** [comment]
Does ~alpha count as less than 2.0.0!? You could just force the last 1.x relase (1.3.1?)?

**Comment by Alasdair** [comment]
Yeah my current hypothesis is "2.0.0~alpha1" is strictly less than "2.0.0". I'm seeing if a "< 1.4.0" bounds gets through the CI action.

Reading the documentation maybe "< 2.0.0~~" is the right version bound but it's not clear
